### ISSUE TITLE
Manually imported subtitle is upgraded

### ISSUE THREAD

**Issue by Bazzu85** [morpheus65535/bazarr#1368]
**Describe the bug**
Hi,
manually imported a subtitle and see that the score was 100%.
After then the upgrade task updated the subtitle from opensubtitle with a score of 99.17% (the same subtitle uploaded by me on the provider).
It's a normal behaviour?
In my head if the subtitle have a score of 100%, no other subtitle have to be grabbed.

**To Reproduce**
check above

**Expected behavior**
check above

**Screenshots**
https://imgur.com/a/AnHwtaK

**Software (please complete the following information):**
 - Bazarr: v0.9.3 
 - Radarr version 
 - Sonarr version 3.0.6.1196
 - OS: Windows 10 pro

**Additional context**
check above

**Comment by morpheus65535** [comment]
Why not simply look into the settings and disable upgrade for manually uploaded subtitles?

**Comment by Bazzu85** [comment]
> Why not simply look into the settings and disable upgrade for manually uploaded subtitles?

Because is already disabled..
![Screenshot_20210405_192132](https://user-images.githubusercontent.com/7345587/113603422-3836f380-9644-11eb-9e08-891f3d63cfa4.jpg)

**Comment by Bazzu85** [comment]
Hi,
additional infos..
1. I added manually the sub for The.Flash.2014.S07E02.1080p.WEB.H264-CAKES[rartv]
2. Checked the option to upgrade manually uploaded subtitles is disabled
3. for test, enabled, saved, disable, saved
4. restarted bazarr
5. started the Upgrade previously downloaded Subtitles task
6. subtitle upgraded from opensubtitle (fortunately was the same because I previously uploaded it)
7. attacched the debug log
[bazarr (4).log](https://github.com/morpheus65535/bazarr/files/6260056/bazarr.4.log)
